tests/test_api.py

```diff
@@ -530,8 +530,8 @@ def test_create_worker_build_ioerror(self, osbs):  # noqa
         )
     )
     @pytest.mark.parametrize(('platforms', 'raises_exception'), (
```

Comment:
Any reason to keep `raises_exception` parameter if it's always `False`?

Reply:
Fixed.